services/nycrc/nycrc.tester.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+const { isIntegerPercentage } = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('valid .nycrc')
+  .get('/yargs/yargs.json?config=.nycrc')
+  .intercept(nock =>
+    nock('https://api.github.com')
```

Comment:
We tend to use real data as much as possible, especially for the "happy case" tests. This helps us spot changes in behaviour/performance of upstream APIs early on. Could `nock` be removed from this test and the `package.json with nyc stanza` one?

Reply:
I've switched the happy paths to hit the real `yargs/yargs` and, based on your suggestion added a test that hits `badges/shields`.

I left a couple `nock` tests for the unhappy paths 😄